src/components/Page.vue

```diff
@@ -112,6 +114,15 @@ export default {
       this.rescan();
     },
 
+    /**
+     * Action after clicking on "Clear" button of search box.
```

Comment:
Just suggestion: I think mention that is _Action after clicking on "Clear" button_  can be omit. In description you should describe what this method doing.

Reply:
Nice, I will enhance description in next pull request